src/infrastructure/config.ts

```diff
@@ -216,3 +216,7 @@ export function isTest() {
 export function isProduction() {
   return configScope.isProduction()
 }
+
+export function decodeJwtConfig(jwtPublicKey: string) {
```

Comment:
🟢 I don't think it needs to be exported

Reply:
probably would be much harder to test otherwise

Reply:
Good point, nevermind 😄